docs/_tutorials/publish-subscribe.md

```diff
@@ -151,13 +133,20 @@ sessionFuncInfo.eventInfo.user_p = NULL;
 
 /* Configure the Session properties. */
 propIndex = 0;
+vpnname = argv[2];
+username = strsep(&vpnname,"@");
```

Comment:
Can we be consistent in the args parsing. Perhaps use char * for all args?

Reply:
We can use char * for all for sure. This is just necessary because the username and vpn name are read as a single parameter and need to be split. I'll make the change